my-app/src/components/cart-card.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
Just a suggestion, name of this component could be cart-item, as it's easier to understand the hierarchy, i.e, there are cart items inside a cart. As your cart item can be made of a card/modal or some other html element.